otj-metrics-actuator/src/main/java/com/opentable/metrics/actuate/micrometer/OtMicrometerToDropWizardExportConfiguration.java

```diff
@@ -58,18 +59,22 @@
  *
  */
 @Configuration
-@ConditionalOnProperty(prefix = OtMicrometerToDropWizardExportConfiguration.CONFIGURATION_PREFIX, name = "enabled", havingValue = "true", matchIfMissing = false)
+@Import(OtMicrometerConfig.class)
+@ConditionalOnProperty(prefix = OtMicrometerConfig.CONFIGURATION_PREFIX, name = "enabled", havingValue = "true", matchIfMissing = false)
```

Comment:
And this is whether it's on at all? I'm having trouble understanding the logic, so I take comfort in knowing in this PR you'll have updated the README

Reply:
Added notes to CHANGELOG